docs/source/configure.rst

```diff
@@ -96,6 +96,32 @@ Batch, BYOC, and other APIs). There is "OAuth clients" frame where we can create
     For detailed instructions on how to obtain credentials, you can see the `Sentinel Hub webinar`_.
 
 
+Copernicus Data Space Ecosystem Configuration
+*********************************************
+
+
+For Copernicus Data Space Ecosystem users, please follow the `Sentinel Hub Services Authentication instructions`_ to
+register an OAuth client using the `Sentinel Hub Services Dashboard`_.
+
+With the registered OAuth client, a valid Copernicus Data Space Ecosystem configuration should be configured as below:
+
+.. code-block:: python
+
+    from sentinelhub import SHConfig
+
+    config = SHConfig()
+    config.sh_client_id = 'oauth-client-id'
+    config.sh_client_secret = 'oauth-client-secret'
+    config.sh_base_url = 'https://sh.dataspace.copernicus.eu'
+    config.sh_token_url = 'https://identity.dataspace.copernicus.eu/auth/realms/CDSE/protocol/openid-connect/token'
+
+
+.. admonition:: Supported Data Collections
+
```

Comment:
here I'd also mention that when using `DataCollections` they also need to be configured for CDSE and then a link to the appropriate example notebook.

Reply:
It will be a bit tricky to mention it here. In short, APIs with a dedicated class, e.g., catalog, batch, batch stat, etc., do not require a specific `DataCollection.define_from()` configuration. Other requests built on `SentinelHubServices`, `SentinelHubFeatureIterator` and `TileSplitter` require the special configuration.

To  avoid confusion, I'd keep CDSE as a separate session and direct CDSE to follow examples there.

Reply:
I was thinking something along the lines of 
```
 With the Copernicus Data Space Ecosystem Configuration, please find the available data collections on
    `Copernicus Data Space Ecosystem indexed in Sentinel Hub`_. See [the example notebook](link-to-notebook) on how to configure them for direct use in `sentinelhub-py`.
```

or do you think its too specific to process API and the likes?

Reply:
How about this?
```
See [the Copernicus Data Space Ecosystem tutorials](link-to-cdse-index) on how to configure them for direct use in `sentinelhub-py`.
```

Reply:
sounds good

Reply:
Added